source/agg-pipeline-builder/view-pipeline-explain-plan.txt

```diff
@@ -0,0 +1,74 @@
+.. _view-pipeline-explain-plans:
+
+=================================
+View Explain Plans for a Pipeline
+=================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To help you better understand the performance of your pipeline, you can view 
+your pipeline's explain plan. You can view the explain plan at any point while 
+creating or editing your pipeline.
+
+About this Task
+---------------
+
+On the :guilabel:`Explain` modal, you can view the explain stages as a 
+:guilabel:`Visual Tree`, where each stage of the pipeline appears as a node on 
+the tree. Alternatively, you can view the explain details in raw JSON format. 
```

Comment:
(suggestion)
For parallelism, consider including where you view the explain results in raw JSON?